src/jira/client/axios.ts

```diff
@@ -147,14 +147,18 @@ const instrumentRequest = (response) => {
 const instrumentFailedRequest = (baseURL: string, logger: Logger) => {
 	return async (error: AxiosError) => {
 		instrumentRequest(error?.response);
-		if (error.response?.status === 503) {
+		if (error.response?.status === 503 || error.response?.status === 405) {
 			try {
 				await axios.get("/status", { baseURL });
 			} catch (e) {
 				if (e.response.status === 503) {
 					logger.info(`503 from Jira: Jira instance '${baseURL}' has been deactivated, is suspended or does not exist. Returning 404 to our application.`);
 					error.response.status = 404;
 				}
+				if (e.response.status === 302) {
```

Comment:
nit: use `else/if` instead of `if`

Reply:
If I use `else if` I will have to define another `else` at the end, but there's nothing else I want to do there. Do I just do else { return; } ?

Reply:
`else if` doesn't force to have `else` at the end, so skip `else` if not required

Reply:
Ooo nice I didn't know that, I though `else` was required. Thanks Harminder, just updated :)